core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3234,6 +3234,41 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/fileTransfers:
+    get:
+      tags:
+        - workspaces
+      summary: List pending file transfers for this workspace
```

Comment:
It took me a minute to grok the semantics of this endpoint wrt incoming vs outgoing -- would it make sense to reword slightly to indicate that these are _incoming_ file transfers?

Reply:
I agree, it's too vague. I've updated the description in the latest commit